src/liquid/components/ld-button/ld-button.css

```diff
@@ -6,7 +6,7 @@
   }
 }
 
-.ld-button {
+:where(.ld-button) {
```

Comment:
Using `:where` might be dangerous here because of reset styles. We should at least have a specificity of 1.

Reply:
Are you sure about this or is this just "playing it safe"? I ask, because the `margin: 0` is causing issues here.

Reply:
I'm pretty sure about this as I had an issue with Tailwind's preflight.